examples/datasets/plot_oldfiles_set_montage.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+.. _plot_montage:
+
+Setting up a standard montage template into an old file
+=======================================================
```

Comment:
@massich maybe we don't need this example no?

Reply:
That was my MWE to understand what was going on and I just put some more stuff around it to make an example out of it. Just in case get into those sub-set errors etc.. So that we can point somewhere. 

But I agree that most likely this is YAGNI. So I'm fine killing it. Unless somebody finds it useful and points out how should it be completed. But I did it at a very early stage of the PR. It might be completely useless now.

Reply:
You can just move it to a `gist` and link to it in a comment in the code

Reply:
thats a great idea! I did not think about it.

Reply:
I agree this example is confusing